src/util/nodeSelectors.ts

```diff
@@ -54,6 +54,23 @@ export function simpleSelectionExtractor(
   };
 }
 
+export function pairSelectionExtractor(
```

Comment:
Docstring would be great here as it's not obvious to me what this function is supposed to do based on its name

Reply:
done